shadowsocks-csharp/Util/ViewUtils.cs

```diff
@@ -25,7 +25,7 @@ public static class ViewUtils
         public static IEnumerable<MenuItem> GetMenuItems(Menu m)
         {
             if (m?.MenuItems == null || m.MenuItems.Count == 0) return Enumerable.Empty<MenuItem>();
-            var children = new List<MenuItem>();
+            var children = new List<MenuItem>(m.MenuItems.Count);

```

Comment:
maybe some LINQ will solve this easier?

Reply:
I tried bit it does not work for MenuItems